### DISCUSSION TITLE
Installation failed because of ffpyplayer build failure

### DISCUSSION THREAD

**Discussion by rechtem** [kivy/kivy-github-discussions#42] [Q&A]
Hello. I know this is not the responsibility of Kivy, but some may have experimented this problem and found a workaround.
I am on Arch (aarch64) on a Rpi 3, and recently FFMPEG moved to v7.0.
The building of ffpyplayer fails because of a deprecated API use. See https://github.com/matham/ffpyplayer/issues/166
Thanks

**Comment by akshayaurora** [comment]
This is not an issue tracker. Please use this forum only for discussions related to the framework , not for issues related to particular repository
